.fixtures.yml

```diff
@@ -6,6 +6,7 @@ fixtures:
     trusted_ca:        "https://github.com/jlambert121/jlambert121-trusted_ca"
     concat:            "https://github.com/puppetlabs/puppetlabs-concat"
     apache:            "https://github.com/puppetlabs/puppetlabs-apache"
+    java_ks:           "https://github.com/puppetlabs/puppetlabs-java_ks"
```

Comment:
Interesting that we use the puppetlabs modules for tests but also host our copies. e.g If production releases include https://github.com/theforeman/puppetlabs-apache then we should probably run tests with that. from katello-installer or foreman-installer spec its not clear what the source of these are.

Do we need to make a copy of https://github.com/theforeman/puppetlabs-java_ks in theforeman namespace?

Reply:
We've only used our forked puppetlabs-apache when puppetlabs was too slow with releases but luckily they've started to pick up speed there so we're on regular pl-apache releases.